api/PasswordCredential.json

```
@@ -24,7 +25,11 @@
           "opera": "mirror",
           "opera_android": "mirror",
           "safari": {
-            "version_added": false
+            "version_added": false,
+            "impl_url": [
+              "https://webkit.org/b/167375",
+              "https://webkit.org/b/252908"
+            ]
```

Dito
```suggestion
            "version_added": false,
            "impl_url": "https://webkit.org/b/252908"
```

PS: I left a comment in the WebKit bug tracker to reflect it as a dependency on the meta bug.